Core/ClientSMLSWIG/Python/pyproject.toml

```diff
@@ -123,6 +123,11 @@ skip = [
     # which would be a duplicate with the x86 runner.
     "cp38-macosx_arm64"
 ]
+# We only need one target to build on, since we test all versions later.
+#
+# We pick python 3.9, since macOS ARM64 doesn't have 3.8, and so this is the earliest version on all platforms.
+build = "cp39*"
+environment = { SOAR_PYTHON_ABI3="1" }
```

Comment:
Does this mean that it *is* enabled by default for all users? Or is this only used in CI? (comment somewhere would be helpful, if there isn't one already)

Reply:
Only in CI, since this is inside cibuildwheel's configuration, but i'll add a comment.